types/firestore.d.ts

```diff
@@ -1533,6 +1533,18 @@ declare namespace FirebaseFirestore {
     | 'not-in'
     | 'array-contains-any';
 
+  /**
+   * InformationalQueryPlan contains information about the query plan.
+   * Contents are subject to change.
```

Comment:
Wondering if all readers will get the point of this comment, maybe we need to go further and explain what that means for someone coding against this type.

Reply:
> How would you recommend expanding the above comment?

Maybe something like this:

```
  /**
   * InformationalQueryPlan contains information about the query plan. This should be inspected or logged,
   * because the contents are intended to be human readable. Contents are subject to change and it is advised
   * to not program against this object.
```

Reply:
Done